enterprise/api/v1/views/enterprise_customer_sso_configuration.py

```diff
@@ -59,3 +78,115 @@ def oauth_orchestration_complete(self, request, configuration_uuid, *args, **kwa
         sso_configuration_record.active = True
         sso_configuration_record.save()
         return Response(status=HTTP_200_OK)
+
+    @permission_required(
+        'enterprise.can_access_admin_dashboard',
+        fn=lambda request, *args, **kwargs: get_customer_from_request(request)
```

Comment:
Do we need lambda here?
```suggestion
        fn=get_customer_from_request
```

Reply:
I believe it does? I don't believe `permission_required` is smart enough to grab the customer uuid from the request object on its own.

Reply:
just passing in that custom made get_customer_from_request isn't enough, so unless you can think of a way

Reply:
Hmm, can we at least remove the *args, **kwargs arguments in the lambda that aren't used?

Reply:
FWICT it's expected to take in the same params as the wrapped method, and the endpoints subclass django mixins that expect args and kwargs as params. So I can't really find a way to do it without having to ignore a linter rule somewhere